document/core/syntax/values.rst

```diff
@@ -145,14 +145,17 @@ Conventions
 
 * The meta variable :math:`z` ranges over floating-point values where clear from context.
 
-.. index:: ! simd, fixed-width, vector
-   pair: abstract syntax; packed simd number
-.. _syntax-simd:
 
-Fixed-Width SIMD
-~~~~~~~~~~~~~~~~
+.. index:: ! vector numbers, integer, floating-point, lane, SIMD
+   pair: abstract syntax; vector
+.. _syntax-vecnum:
+
+Vector Numbers
```

Comment:
Vector Numbers sound a bit funny to me. Can we consider any other names? How about, Vector of numbers or Numeric vectors?

Reply:
Yes, that sounds better. Changed.